test/Npgsql.Tests/PrepareTests.cs

```diff
@@ -70,6 +70,24 @@ public void Unprepare()
             }
         }
 
+        [Test]
+        public async Task UnprepareAsync()
```

Comment:
Let's merge this into Unprepare, and have the test accept an async flag to reduce code duplication.